.azure-pipelines/pipelines.yml

```diff
@@ -59,7 +59,10 @@ stages:
               GCP_SERVICE_ACCOUNT_KEY: $(GcpServiceAccountKey)
             displayName: "Generate docs"
 
-          - script: ci/run_envoy_docker.sh 'ci/upload_gcs_artifact.sh /source/generated/docs docs'
+          - script: |
+              SHORT_COMMIT_SHA=$(git log --pretty=%P -n 1 | cut -d' ' -f2 | head -c7)
+              export SHORT_COMMIT_SHA
```

Comment:
Move this part into upload_gcs_artifact.sh, and make it consistent with coverage report as well.

A fixed URL is desired for latest main branch coverage report, currently at https://storage.googleapis.com/envoy-postsubmit/master/coverage/index.html

Reply:
im afraid im really not following. 

> A fixed URL is desired for latest main branch coverage report, currently at https://storage.googleapis.com/envoy-postsubmit/master/coverage/index.html

im really not following. 

the `SHORT_COMMIT_SHA` is only set when uploading docs, how  will that affect the postsubmit coverage report  ?

Reply:
i moved setting the sha/path into the upload script

im guessing re coverage, you just meant dont mess up the path for that (i dont think it does/did)